# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/.cargo-checksum.json b/third_party/rust/d3d12/.cargo-checksum.json
--- a/third_party/rust/d3d12/.cargo-checksum.json
+++ b/third_party/rust/d3d12/.cargo-checksum.json
@@ -1,1 +1,1 @@
-{"files":{"CHANGELOG.md":"6b35a919da86138b9546a8de29366ec0beebbb41833a28d01f67c213457bf3ea","Cargo.toml":"2f19cd78b162d8ee0c27b81a1a2ceff681f5b9e933dd9ddb1d59199dae323810","README.md":"4547821e043cf24e53f94519248ca13813a9cd6e55c6b23dc4273c502e0155b0","appveyor.yml":"14da30f5712e0eaec1acbb74649f391597095d8f8aaccf7a528a358936e8f98b","bors.toml":"31d348faf24d2bac765198bce62b4c4f1d6987e567eaed3d380a6704fec5183c","rustfmt.toml":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","src/com.rs":"a7f2dfc82064a580deb8f7a2b67ca95152fb3225d41548680b362fdd28b7f706","src/command_allocator.rs":"decd4f38070799e7c6e8a74334717ed858096d1b04677b4e673ae58ea00afec3","src/command_list.rs":"b2288f38db7c942fba7d57577410e43ded95de6e48517e793d5585c998bd0f61","src/debug.rs":"56d16855d6959ab0c7195fec480ab2385350bc4b973120a502b21cfc2bc2e656","src/descriptor.rs":"15fba309ef71c9e9ae6f1f1492f7f4d1ed1e7e9f7ed225c8c11f1e7b5512cda1","src/device.rs":"b5270f06f33d0ac4a7ad92383b9fda694129ad4235c8f01c9e6a344375994e7e","src/dxgi.rs":"879878fb59ffbecb5ec0008a2e8133b46f1625d73346f8f9127c0ae8de4fe7be","src/heap.rs":"a317bebf043b8b8c4a2030745b36b28f1c56f2afe7c2ef86bf5225d5d475804e","src/lib.rs":"b5a158ad4613a2b132f24b63f96908cb21d1e012f936ee9c6d5d59fe27811733","src/pso.rs":"7a22633fe4afa75d47464ff9f0ae0b0d96862dfbc82c41c844d0a78557e680cd","src/query.rs":"88f41457080e99b3915bda1c47b6a85f738985f438a9104d2ec774fc0e56c106","src/queue.rs":"1ae3372dbf5010c9fad4260c0ff9afbaf7f1c24ea5e3b2890508f5994f50e521","src/resource.rs":"a0c9bca3a6db3fc89222323e238b386055ae74ef733d06aafdc24f934610ae48","src/sync.rs":"00e597eb6561050ffa4a0dec9f25b6af9b23e32ac0d5c3efaa9d8c8eadb72431"},"package":"091ed1b25fe47c7ff129fc440c23650b6114f36aa00bc7212cc8041879294428"}
\ No newline at end of file
+{"files":{"CHANGELOG.md":"c0ed28ff370cd6fdb2d83b135e4bb0249b1fe5c708c82118cf76aedf301c6e90","Cargo.toml":"a28c00d11cfe7a3164b1eafb5c5c93b97533f51b498ce666e3c8932da42d310a","README.md":"71f2c62c9f9a892b436adf130dab47348744ea05c98af95599d2e79b54fb25a5","appveyor.yml":"69e6279a533b60e4f7ba70e645a9c6b7aba1654105a1e1362e67ed14deca5368","bors.toml":"366ea95cdc64dae238edd4fe70a3d5f698b4dd22064abeb8d91550c81c8ccc67","rustfmt.toml":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","src/com.rs":"1610750dcf9bd9b4ca469d37675ddbad286880e2b187952df4af0b83641bf4c7","src/command_allocator.rs":"b430fa99c7c541f5e592d6e9b8c83265b44f3f1b70205f9d44b4519e2d161cea","src/command_list.rs":"95ee2e40b6af96f3ac7471f4455e4121b4e13618d2d36f4ca7fa58887cbacdb6","src/debug.rs":"6d04c96fa2073ca8e3a4c29088c4f654c9bbd4944021196a209166ecf21520f9","src/descriptor.rs":"45a4fd8a2d5b870b9754c54032d920774cbfcb5a6e29ac644117ffa1ad5b3eff","src/device.rs":"b4ac053d9a85d070d049eac7f243edae7afceb5c9f6d75ae1faddc2ec2875ca9","src/dxgi.rs":"36251ec2d55009796070df53591b3129e1eccadeeb0442818bc5d81065305050","src/heap.rs":"bb4c0996c63da3dc14366aaa23068e7a3d2fb43d617f8645a5aef74767a463d6","src/lib.rs":"21b72a9ef5ee0f4ad1fb1879e6e117568804de7ed01933739274e48a1c0d324d","src/pso.rs":"1dcf102f061a3cadfc0de3fd75e2414f06c1bf9ac5727be1cbdd2204883093e4","src/query.rs":"53f64ef6f2212a1367f248191e4ab93a3facb18c37709f05f850c30bdc7be8cf","src/queue.rs":"3cd807b1df00ef9dd6ba5a28dcee883033ea174d3715b6de754c6f890159302a","src/resource.rs":"9a0f53f8f23fd671ae44370413274606ce62942bb16fc7370e6f32a2410b4255","src/sync.rs":"dcce20cfd2a408ad43ad6765a91d65dfe1998615cb56a3cfb60306094f2365a8"},"package":"2daefd788d1e96e0a9d66dee4b828b883509bc3ea9ce30665f04c3246372690c"}
\ No newline at end of file